src/reusable-blocks/class-reusable-blocks-translation.php

```diff
@@ -24,14 +24,20 @@ public function __construct( \SitePress $sitepress ) {
 	 */
 	public function convert_block( array $block, $lang = null ) {
 		if ( Reusable_Blocks::is_reusable( $block ) ) {
-			$block['attrs']['ref'] = $this->sitepress->get_object_id(
-				$block['attrs']['ref'],
-				self::POST_TYPE,
-				true,
-				$lang
-			);
+			$block['attrs']['ref'] = $this->convert_block_id( $block['attrs']['ref'], true, $lang );
 		}
 
 		return $block;
 	}
+
+	/**
+	 * @param int         $block_id
+	 * @param bool        $original_if_missing
+	 * @param string|null $lang
+	 *
+	 * @return
+	 */
+	public function convert_block_id( $block_id, $original_if_missing, $lang = null ) {
```

Comment:
@strategio I would remove `$original_if_missing` because I think you always want to call it with true.